app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Api::V1::UsersController < ApplicationController
+  # GET /users
+  def index
+    @users = User.all
+
+    render json: @users
+  end
+
+  # GET /users/1
+  def show
+    render json: @user
+  end
```

Comment:
- Please the func to find the reservation before rendering. 🐱

Reply:
done :+1: